Dockerfile

```diff
@@ -2,6 +2,16 @@ FROM registry.access.redhat.com/ubi8/ubi:latest AS build-stage0
 ARG OC_VERSION="stable"
 ENV OC_URL="https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/${OC_VERSION}"
 
+# AWS cli args and env variables
+# Replace AWS client zipfile with specific file to pin to a specific version
+# (eg: "awscli-exe-linux-x86_64-2.0.30.zip")
+ARG AWSCLI_VERSION="awscli-exe-linux-x86_64.zip"
+ENV AWSCLI_URL="https://awscli.amazonaws.com/${AWSCLI_VERSION}"
```

Comment:
Shall we install this with EPEL? I think it is better

Reply:
is it available in EPEL? I mixed official docs and ocm-container's method of installation of the aws cli

Reply:
I guess this is what you wanted
https://aws.amazon.com/premiumsupport/knowledge-center/ec2-enable-epel/
but I am not sure how good that is